v2/spiffex509/svid.go

```diff
@@ -0,0 +1,23 @@
+package spiffex509
+
+import (
+	"crypto"
+	"crypto/x509"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+// SVID represents a SPIFFE X509-SVID.
+type SVID struct {
```

Comment:
maybe a convenience field for getting the trust domain that the user is in?

Reply:
That sounds helpful.

Reply:
Actually, that's already there. `svid.ID.TrustDomain()`